### ISSUE TITLE
OpenAPI spec not valid?

### ISSUE THREAD

**Issue by jvolkening** [ncbi/datasets#289]
Hello,

In my hands the published OpenAPI spec for Datasets (and also the latest version in git) fails validation. This appears to be caused by empty values for `example` `summary` keys, e.g.

```yaml
        examples:
          example-0:
            value: accession
            summary: 
```
I initially ran into this problem trying to use the YAML spec in an API generator, and then tried to load it in the [Swagger Editor](https://editor.swagger.io/) to check the validation and saw similar errors, e.g.

```
Structural error at paths./genome/accession/{accessions}/dataset_report.get.parameters.17.examples.example-0.summary:
should be string
```

When I used a quick script to replace all 260 or so of these empty values with 'none', the resulting file passed validation. Unfortunately the script did not preserve other formatting of the YAML so I can't easily upload these changes as a PR.

**Comment by syntheticgio** [comment]
Thanks @jvolkening; this is a bit strange since it has been like this for over a year; but I will pass it along.  I believe this file is autogenerated so we may have to dig into what is causing it in the pipeline.  I'll leave the ticket open while we investigate; but you're not currently blocked on your work by this correct?

**Comment by jvolkening** [comment]
> you're not currently blocked on your work by this correct?

Thanks -- no, this is just Day 1 of investigating migrating our tools to the Datasets approach, based on the announced May 2024 transition.

**Comment by jvolkening** [comment]
> If you're referring to the May 2024 deprecation of the Assembly and Genome web interfaces, Eutils and FTP will still work (if that helps).

That's good to know, thanks -- right now we mainly use esearch+efetch or pull larger genomes from FTP after parsing the `assembly_summary.txt` files, and I thought they were going away too. I still imagine the Datasets API may be more efficient for some of the things we do regularly.

**Comment by syntheticgio** [comment]
Hi @jvolkening - I'm being told this should have been fixed a few months ago (sorry for the late notice, I must have missed it internally).  I'm going to close the issue, but if you are still seeing the problem on your end feel free to open a new issue or reopen it.
